src/UpdateAssemblyInfo/Program.cs

```diff
@@ -113,7 +112,7 @@ private static string GetGitPath()
                 }
             }
 
-            var gitPath = Path.Combine("Git", "bin", "git.exe");
+            var gitPath = "Git\\bin\\git.exe";

```

Comment:
I think it's just nicer to always use the platform independent Path.Combine method